src/less/navbar.less

```diff
@@ -17,7 +17,7 @@
   &:extend(.clearfix all);
 
   @media (min-width: @grid-float-breakpoint) {
-    border-radius: @navbar-border-radius;
+    .border-radius(@navbar-border-radius);
```

Comment:
@osbornm What do you suggest we do? Remove the mixin and replace everything with `border-radius: value`? Just update the mixin to only spit out `border-radius: value`?